docs/mast/mast.rst

```diff
@@ -834,6 +885,15 @@ Given an HSC Match ID, return all catalog results.
    410574498 63980492 ...   -1.10056e-005 1.56577e-009 1.56577e-009 1.10056e-005
    410574497 63980492 ...   -1.10056e-005 1.56577e-009 1.56577e-009 1.10056e-005
 
+"MatchID" is case sensitive in this case, and will produce an error with different
```

Comment:
I'd remove this example, since I think most users know that the metadata is case-sensitive.